cfgov/unprocessed/js/molecules/Notification.js

```diff
@@ -133,6 +133,10 @@ function Notification( element ) { // eslint-disable-line max-statements, inline
     return this;
   }
 
+  function getElement() {
```

Comment:
I'm saying why not just expose it with a property like: 

`this.element = _dom`